packages/serve/src/startDevServer.ts

```diff
@@ -10,6 +8,13 @@ import Server from 'webpack-dev-server/lib/Server';
  * @returns {Void}
  */
 export default function startDevServer(compiler, options): void {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let Server: any;
+    try {
+        Server = require('webpack-dev-server/lib/Server');
+    } catch (err) {
+        throw new Error(`You need to install 'webpack-dev-server' for running 'webpack serve'.\n${err}`);
+    }
```

Comment:
I think it is unnecessary, we already do check before

Reply:
We are importing this file before the `serve` function, so I think it is kind of necessary as well. Am I missing something?

Reply:
We run `startDevServer` after first check so we don't need it :smile:

Reply:
Yeah, my bad. Updated now.